setup.py

```diff
@@ -41,7 +37,12 @@ def grep(attrname):
     license='Apache 2.0',
     packages=['arrow'],
     zip_safe=False,
-    install_requires=install_requires,
+    install_requires=[
+        'python-dateutil',
+        ],
```

Comment:
Could you just fix this alignment? Just remove 4 spaces, everything else is LGTM.